include/openssl/stack.h

```diff
@@ -10,6 +10,8 @@
 #ifndef HEADER_STACK_H
 # define HEADER_STACK_H
 
+# include <stddef.h>
```

Comment:
Why is this needed?

Reply:
I started allowing an expansion with a size_t argument but decided that was insane because the count is an int.